### PR TITLE
[Filebeat] Add dashboards to CEF module

### PR DESCRIPTION
**Pull request by andrewkroh**
This migrates the dashboards from the Logstash ArcSight module into the Filebeat CEF module.

*Breaking Change:* The mapping for the `cef.extension.*` fields was changed from all `keyword` types to match the data types defined in the CEF guide. The `decode_cef` processor will convert the strings to the correct data type. If any value does not convert then that field is dropped and `error.message` is updated with info about the failure.

And because of the limit on field expansion in Elasticsearch queries the `cef.extension.*` fields are excluded from `default_field` in Filebeat's index template.

### CODE REVIEW

File: x-pack/filebeat/processors/decode_cef/cef/types.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+// you may not use this file except in compliance with the Elastic License.
+
+package cef
+
+import (
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
```
Comment (adriansr) [inline x-pack/filebeat/processors/decode_cef/cef/types.go]: Nit: This should go after github.com/pkg/errors

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -401,6 +402,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `index` option to all inputs to directly set a per-input index value. {pull}14010[14010]
 - Remove beta flag for some filebeat modules. {pull}14374[14374]
 - Add attack_pattern_kql field to MISP threat indicators. {pull}14470[14470]
+- Add dashboards to the CEF module (ported from the Logstash ArcSight module).
```
Comment (adriansr) [inline CHANGELOG.next.asciidoc]: Dupe

File: x-pack/filebeat/processors/decode_cef/cef/types.go (hunk 1 above)
Comment (andrewkroh) [reply x-pack/filebeat/processors/decode_cef/cef/types.go]: Fixed.

File: CHANGELOG.next.asciidoc (hunk 2 above)
Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]: Removed

**Comment by MarkSettleES** [comment]
@jamiesmith, I wanted to bring this to your attention because I recall getting your help last year with a screenshot of a dashboard of CEF-formatted data and figure that you _might_ find it helpful to know.

**Pull request merged**